lib/FunctionIsolation/EnforceABI.cpp

```diff
@@ -801,17 +883,19 @@ void EnforceABIImpl::generateCall(IRBuilder<> &Builder,
     Callee = IndirectPlaceholderPool.get(NewType,
                                          NewType,
                                          "indirect_placeholder");
+
+    Result = Builder.CreateCall(Callee, Arguments);
   } else {
 
     // Additional debug checks if we are not emitting an indirect call.
     revng_log(EnforceABILog,
               "Emitting call to " << getName(Callee) << " from "
                                   << getName(CallSite.Call));
 
-    FunctionDescription &Function = FunctionsMap.at(Callee);
-    revng_assert(Function.Function != nullptr
-                 and Function.Function->getName().startswith("bb."));
-    if (GlobalVariable *CSV = CallSite.isCompatibleWith(Function)) {
+    FunctionDescription &Func = FunctionsMap.at(Callee);
```

Comment:
Don't abbreviate

Reply:
I abbreviated to `Func`, because `Function` was creating a name collision with `llvm::Function` over there (after the new changes).

Reply:
Find a more precise name or go for `TheFunction`. Abbreviation make the code come from the 90 and harder to read.